test-five/frontend/Dockerfile.dev

```diff
@@ -0,0 +1,17 @@
+# Use an official Node.js runtime as the base image
```

Comment:
can you explain why you chose to introduce docker here?

Reply:
I have responded to this question [here](https://github.com/moshtix/show-us-what-you-got/pull/69/files/892c3f395fc63cb15c59cc1e4e640bfc04320e5d#r1378331991). I should have removed all Docker references as it's not used in the project, serving the sole purpose of local development.